internal/rbd/rbd_util.go

```diff
@@ -98,11 +98,10 @@ type rbdVolume struct {
 	MonValueFromSecret  string `json:"monValueFromSecret"`
 	VolSize             int64  `json:"volSize"`
 	DisableInUseChecks  bool   `json:"disableInUseChecks"`
-	Encrypted           bool
 	readOnly            bool
 	Primary             bool
 	ThickProvision      bool
-	KMS                 util.EncryptionKMS
+	encryption          *util.VolumeEncryption
```

Comment:
so the only concern I have here is that now we're removing "encrypted" flag from the object which means - if any kms configured for the rbdVolume - consider it encrypted... so is it backward compatible with previously created volumes and metadata they have already?

Reply:
I guess it should be since if kms id is found in volume metadata - we create volume encryption instance I think and go from there... but want to double check really

Reply:
Yes, it is backwards compatible. The on-disk structure does not change, this is the main `rbdVolume` struct, and it does not have any on-disk representation.